src/software/pipeline/main_panoramaWarping.cpp

```diff
@@ -1094,8 +1101,9 @@ int aliceVision_main(int argc, char **argv)
   // If panorama width is undefined, estimate it
   if (panoramaSize.first <= 0)
   {
+    float ratioUpscale = float(percentUpscale) / 100.0f;
```

Comment:
We may need to clamp to 0-1 to avoid crashes.

Reply:
will do